mlrun/datastore/store_resources.py

```diff
@@ -141,7 +141,7 @@ def _get_store_resource(uri, use_cache=True):
         return _get_store_resource
 
 
-def get_store_resource(uri, db=None, secrets=None, project=None):
+def get_store_resource(uri, db=None, secrets=None, project=None, store_secrets=None):
```

Comment:
I just realized that my suggestion was bad here - the function is called `get_store_resource` where `store` refers to the MLRun artifact store... So, adding a `store_secrets` parameter is bad naming since it may confuse the data-store with the MLRun store.
So, let's rename the parameter to `data_store_secrets` to make the distinction clear.